src/main/java/api/clients/ApiListClient.java

```diff
@@ -28,15 +28,30 @@ public ListResponse createNewList(ListBuilder listBody, String idBoard, int expe
     }
 
     @Step("Rename created list on board with id: {idList}. New name of list {listName}. Expected status code {expectedStatusCode}")
-     public ListResponse renameList(String listId, String listName, int expectedStatusCode){
+    public ListResponse renameList(String listId, String listName, int expectedStatusCode) {
         return given()
                 .spec(requestSpec)
-                .queryParam("name",listName)
-                .put("/1/lists/{id}",listId)
+                .queryParam("name", listName)
+                .put("/1/lists/{id}", listId)
                 .then()
                 .statusCode(expectedStatusCode)
                 .log()
                 .body()
                 .extract().as(ListResponse.class);
-     }
-}
+    }
+
+    @Step("Move cards from source list to target list")
+    public ListResponse[] moveCardsToAnotherList(String idBoard, String idListSource, String idListTarget, int expectedStatusCode) {
+        return given()
+                .spec(requestSpec)
+                .when()
+                .queryParam("idBoard", idBoard)
+                .queryParam("idList", idListTarget)
+                .post(format("/1/lists/%s/moveAllCards", idListSource))
+                .then()
+                .statusCode(expectedStatusCode)
+                .log()
+                .body()
+                .extract().as(ListResponse[].class);
```

Comment:
Does this endpoint really return the array?
Try use:
```suggestion
                .extract().as(ListResponse.class);
```

Reply:
Yes, we tried the option you suggested earlier with the mentors, but it didn't work